src/framework/mpe/mpetypes.h

```diff
@@ -32,13 +32,19 @@
 #include <vector>
 
 #include "log.h"
-#include "realfn.h"
 
 namespace mu::mpe {
 // common
 using msecs_t = uint64_t;
-using percentage_t = float; // from 0.0 (0%) to 1.0 (100%)
-constexpr percentage_t PERCENTAGE_PRECISION_STEP = 0.1; // 10%
+using percentage_t = int_fast16_t;
+constexpr percentage_t SINGLE_PERCENT = 100;
+constexpr percentage_t HUNDRED_PERCENTS = SINGLE_PERCENT * 100;
+constexpr percentage_t TEN_PERCENTS = SINGLE_PERCENT * 10;
```

Comment:
I believe that in this list, `SINGLE_PERCENT` should be `ONE_PERCENT`

Reply:
I'm not against, will rename

Reply:
done